app/services/translate_status_service.rb

```diff
@@ -32,6 +40,67 @@ def languages
   end
 
   def content_hash
-    Digest::SHA256.base64digest(@content)
+    Digest::SHA256.base64digest(@source_texts.transform_keys { |key| key.respond_to?(:id) ? "#{key.class}-#{key.id}" : key }.to_json)
```

Comment:
That doesn't sound right. For `content` and `spoiler_text`, the key would just be that. Poll options can be edited, and so can media attachment descriptions.

Reply:
It is the only keys that are transformed. The result is e.g. this:

```json
{
  "content": "<p>This is content</p>",
  "Poll::Option-0": "Option 1",
  "Poll::Option-1": "Option 2",
}
```